pro/src/screens/OfferEducational/OfferEducationalForm/FormNotifications/EmailInputRow/EmailInputRow.module.scss

```
@@ -1,14 +1,26 @@
 @use 'styles/mixins/_rem.scss' as rem;
 @use 'styles/variables/_colors.scss' as colors;
 
-.notification-mail-input {
-  width: 90%;
+.notification-mail {
+  position: relative;
+  &-input {
+    width: 90%;
+  }
 }
 
-.trash-icon {
-  height: rem.torem(24px);
-  width: rem.torem(24px);
-  &:hover {
-    fill: colors.$primary;
+.trash {
+  position: absolute;
+  top: rem.torem(36px);
+  right: rem.torem(8px);
+  &-button {
+    background-color: transparent;
+    border: none;
+    &-icon {
+      height: rem.torem(24px);
+      width: rem.torem(24px);
+    }
+    &-icon:hover {
+      fill: colors.$primary;
+    }
```

```suggestion
    &-icon {
      height: rem.torem(24px);
      width: rem.torem(24px);
    
      &:hover {
        fill: colors.$primary;
      }
    }
```

Ca revient au même mais au moins ca groupe tout :)